spring-cloud-deployer-local/src/test/java/org/springframework/cloud/deployer/spi/local/BoundedRandomPortTests.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.deployer.spi.local;
+
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Christian Tzolov
+ */
+public class BoundedRandomPortTests {
+
+	private AbstractLocalDeployerSupport localDeployerSupport;
+
+	@Before
+	public void setUp() {
+		LocalDeployerProperties properties = new LocalDeployerProperties();
+		properties.getPortBound().setLower(30001);
+		properties.getPortBound().setUpper(30213);
+		localDeployerSupport = new AbstractLocalDeployerSupport(properties) {};
+	}
+
+	@Test
+	public void portTests() {
```

Comment:
It'd be good to do actual test to bind using a real property key/value as that verifies that boot binds things ok.

Reply:
Makes sense, i will add integration test for this. 

Not sure about the Docker mode tough? are we testing this?

Reply:
this was just about testing boot binding and it really have nothing to do with docker.

Reply:
Random ports really doesn't work with docker if you need to expose something. But then we're not using random ports with docker?